Tests/HelperTests/StringExtensionsTests+XCTest.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  StringExtensionsTests+XCTest.swift
+//  SwiftyBot
+//
+//  The MIT License (MIT)
+//
+//  Copyright (c) 2016 - 2019 Fabrizio Brancati.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+//
+//
+//
+//  NOTE: This file was generated by generate_linux_tests.rb
+//
+//  Do NOT edit this file directly as it will be regenerated automatically when needed.
+//
+
+import XCTest
+
+internal extension StringExtensionsTests {
+    static var allTests: [(String, (StringExtensionsTests) -> () throws -> Void)] {
+        return [
+            ("testHasGreetings", testHasGreetings),
+            ("testHasNotGreetings", testHasNotGreetings)
+        ]
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All declarations should specify Access Control Level keywords explicitly.](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)